### ISSUE TITLE
Notification buttons should be larger

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#9237]
**Describe the issue you encountered:** Notification buttons are small a bit. It is sometimes hard to click the button with a mouse.

- Brave Version (revision SHA): 0.15.3
    
- Actual result:
  <img width="223" alt="screenshot 2017-06-04 1 31 32" src="https://cloud.githubusercontent.com/assets/3362943/26755328/df125bd4-48c5-11e7-8fd8-73a98d7556dc.png">

- Expected result:
  <img width="200" alt="screenshot 2017-06-04 1 34 15" src="https://cloud.githubusercontent.com/assets/3362943/26755335/f5d6c97c-48c5-11e7-9111-638511e702e4.png">

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:
